src/views/pages/profile/index.tsx

```diff
@@ -1,5 +1,14 @@
-import { Wrapper } from 'views/components/Wrapper';
+import { CharacterSelection } from './component/Characters';
+import { UserName, ProfileContentWrapper, CreateCharacterWrapper, NameDisplay } from './styled';
 
 export const Profile = () => {
-  return <Wrapper>Profile</Wrapper>;
+  return (
+    <ProfileContentWrapper>
+      <UserName>userName.near</UserName>
+      <CreateCharacterWrapper>
+        <NameDisplay placeholder="Display Name" />
```

Comment:
add property view to exist input component and assign different styles for it (e.g. outlined, underlined)